README.md

```diff
@@ -14,26 +15,20 @@ To add your project, please make a pull request. You'll need to create a data fi
 
 See [CONTRIBUTING.md](CONTRIBUTING.md).
 
-Approvers ([@open-telemetry/docs-approvers](https://github.com/orgs/open-telemetry/teams/docs-approvers))
+Roles: 
+- Approvers: [@open-telemetry/docs-approvers](https://github.com/orgs/open-telemetry/teams/docs-approvers)
+- Maintainers: [@open-telemetry/docs-maintainers](https://github.com/orgs/open-telemetry/teams/docs-maintainers))
+- Learn more about roles in the [community
+  repository](https://github.com/open-telemetry/community/blob/main/community-membership.md).
 
-- [Shelby Spees](https://github.com/shelbyspees), Honeycomb
-
-Maintainers ([@open-telemetry/docs-maintainers](https://github.com/orgs/open-telemetry/teams/docs-maintainers))
-
-- [Steve Flanders](https://github.com/flands), Splunk
-- [Morgan McLean](https://github.com/mtwo), Google
-- [Austin Parker](https://github.com/austinlparker), LightStep
```

Comment:
I removed the lists of names because they were outdated. The link to the group is good enough IMHO.